sqlite_query.c

```diff
@@ -690,3 +723,34 @@ static char * sqlite_text_value_to_pg_db_encoding(sqlite3_stmt * stmt, int stmt_
 		/* There is no UTF16 in PostgreSQL for fast sqlite3_column_text16, hence always convert */
 		return (char *) pg_do_encoding_conversion((unsigned char *) utf8_text_value, strlen(utf8_text_value), PG_UTF8, pg_database_encoding);
 }
+
+/*
+ * Converts int64 frrom SQLite to PostgreSQL string from 0 and 1 only
+ */
+char *int2binstr(sqlite3_int64 num, char *s, size_t len)
+{
+	s[--len] = '\0';
+    do
+		s[--len] = ((num & 1) ? '1' : '0');
+	while ((num >>= 1) != 0);
+    return s + len;
+}
+
+/*
+ * Converts PostgreSQL string from 0 and 1 only to int64 for SQLite
+ */
+sqlite3_int64 binstr2int(const char *s)
+{
+    sqlite3_int64 rc;
```

Comment:
I think it better to init `rc` value here instead of initializing at for loop.    
and please using code style:
```C
for (;;)
{
}
if ()
{
}
```

Reply:
Please ensure all is ok.

Reply:
Thank for updating. Please help me to resolve this thread.